src/View/Helper/FormHelper.php

```diff
@@ -881,12 +881,12 @@ public function label($fieldName, $text = null, array $options = [])
      * - `fieldset` Set to false to disable the fieldset. You can also pass an array of params to be
      *    applied as HTML attributes to the fieldset tag. If you pass an empty array, the fieldset will
      *    be enabled
-     * - `legend` Set to false to disable the legend for the generated input set. Or supply a string
+     * - `legend` Set to false to disable the legend for the generated control set. Or supply a string
      *    to customize the legend text.
-     * @return string Completed form inputs.
+     * @return string Completed form controls.
      * @link http://book.cakephp.org/3.0/en/views/helpers/form.html#generating-entire-forms
      */
-    public function allInputs(array $fields = [], array $options = [])
+    public function allControls(array $fields = [], array $options = [])
```

Comment:
@dereuromark that has been discussed in the issue. https://github.com/cakephp/cakephp/issues/9935#issuecomment-269704851

Fields is saying more than controls. Controls are for selects, radios and are generally what we called "inputs" before as most (but not all) are mapped through the input tag.